source/collection-management/timeseries-collection.yml

```diff
@@ -0,0 +1,63 @@
+description: "timeseries-collection"
```

Comment:
I take it folks decided to punt on writing a collection management spec document for the time being. IMO, it still makes sense to keep these tests under a `tests/` sub-directory. Also, you can create a basic `tests/README.rst` file that explains the tests in this directory use the unified format.

Consider [`versioned-api/tests/README.rst`](https://github.com/mongodb/specifications/blob/master/source/versioned-api/tests/README.rst) as a model. That would also be a good place to note that a certain server parameter or feature flag might be required to enable time-series collections on the server.

Reply:
Yep, no full collection management spec for the time being. Created a `tests/` subdirectory with a small `README`. Since `featureFlagTimeseriesCollection` is now enabled by default, the `README` just specifies that these tests are in the unified test format and require schema version 1.0.